.github/workflows/pr-preview.yml

```
@@ -0,0 +1,159 @@
+          APP_NAME="kcd-pr-${{ github.event.number }}"
+          echo "app_name=$APP_NAME" >> $GITHUB_OUTPUT
+          echo "url=https://$APP_NAME.fly.dev" >> $GITHUB_OUTPUT
+
+      - name: 📝 Setup PR-specific configurations
+        run: |
+          # Copy and modify fly.toml for PR preview
+          cp fly.toml fly.toml.backup
+          
+          # Update app name for PR
+          sed -i 's/^app = .*/app = "${{ steps.app-name.outputs.app_name }}"/' fly.toml
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)